fly-base.nix

```diff
@@ -0,0 +1,17 @@
+#
+# This can be replaced by any method of tracking inputs.
+#   - niv     https://github.com/nmattia/niv
+#   - npins   https://github.com/andir/npins
+#   - Flakes  https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-flake.html 
+#
+# Using this simple purely-Nix shim serves as a placeholder.
+
+let
+  # Tracking https://github.com/fly-apps/nix-base/pull/3
+  rev = "cdd35d244ce3b21bc2910e086da517e361744457";
```

Comment:
Would we be able to extract these values to the app spec TOML as well? Maybe that seems like overkill, but it seems smart to keep all config in one place, and perhaps generate this file inside `.fly` to keep it hidden from mortals.

Reply:
Yes.

There are ways that this could also be entirely hidden, no `.nix` at all. But as discussed previously, the initial implementation works directly with Nix, and work for a tooling that hides the implementation details is "for later".